src/test/java/Calculator/CalculatorTest.java

```diff
@@ -0,0 +1,102 @@
+package Calculator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import java.util.zip.DataFormatException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class CalculatorTest {
+    private Calculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new Calculator();
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = "3 + 2:5", delimiter = ':')
+    @DisplayName("덧셈 테스트")
+    void add(String input, Integer expected) throws DataFormatException {
+        assertThat(calculator.calculate(input)).isEqualTo(expected);
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = "3 - 2:1", delimiter = ':')
+    @DisplayName("뺄셈 테스트")
+    void subtract(String input, Integer expected) throws DataFormatException {
+        assertThat(calculator.calculate(input)).isEqualTo(expected);
+    }
+    
+    @ParameterizedTest
+    @CsvSource(value = "3 * 2:6", delimiter = ':')
+    @DisplayName("곱셈 테스트")
+    void multiply(String input, Integer expected) throws DataFormatException {
+        assertThat(calculator.calculate(input)).isEqualTo(expected);
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = "4 / 2:2", delimiter = ':')
+    @DisplayName("나눗셈 테스트")
+    void divide(String input, Integer expected) throws DataFormatException {
+        assertThat(calculator.calculate(input)).isEqualTo(expected);
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = "2 + 3 * 4 / 2:10", delimiter = ':')
+    @DisplayName("다중 연산 테스트")
+    void calculate(String input, Integer expected) throws DataFormatException {
+        assertThat(calculator.calculate(input)).isEqualTo(expected);
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = "2 + 3 * 4 a 2:a", delimiter = ':')
+    @DisplayName("연산자가 아닌 다른 값이 들어왔을 경우 DataFormatException 테스트")
+    void dataFormatException(String input, String expected) throws DataFormatException {
```

Comment:
```suggestion
    void dataFormatException(String input, String expected) {
```
예외를 잡기 때문에 따로 throws 를 할 필요는 없을 것 같아요